src/rules/no-named-default.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    function checkSpecifiers(type, node) {
```

Comment:
There's no reason to specify the type IMO, as I don't think this code will be extended to account for other types. You should remove the `type` parameter and the call to `bind` below. To make things a little bit clearer, you could simply assign this function to ImportDeclaration directly (like `{ ImportDeclaration: function (node) { ... } }`